### PR TITLE
GEOPY-1434: pushing back drillholegroup back using monitoring directory + drag and drop uijson raise errors

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-1434 - pushing back drillholegroup back using monitoring directory + drag and drop uijson raise errors**

### CODE REVIEW

File: geoh5py/io/h5_reader.py [hunk 1]
```diff
@@ -181,21 +181,23 @@ def fetch_concatenated_values(
                 group = h5file[name][entity_type][as_str_if_uuid(uid)][
                     "Concatenated Data"
                 ]
-                if label not in group["Index"]:
+                indices = group["Index"].get(label.replace("/", "\u2044"))
+                if indices is None:
                     return None
```
Comment (MatthieuCMira) [inline geoh5py/io/h5_reader.py:187]: strange to change "/" by unicode but trusting you on this one

**Review by MatthieuCMira — APPROVED** [review]
Seems good for me, going to test it on petro-lingo

**Review by MatthieuCMira — CHANGES_REQUESTED** [review]
I tested it on petro lingo.

I don't have the error message anymore but the data are not being pushed back on the original workspace.

File: geoh5py/io/h5_reader.py [hunk 2]
```diff
@@ -181,21 +181,23 @@ def fetch_concatenated_values(
...
 
-                if label in group["Data"]:
-                    attribute = group["Data"][label][:]
-                else:
-                    attribute = group[label][:]
+                array = group["Data"].get(label.replace("/", "\u2044"))
+                if array is None:
+                    array = group.get(label.replace("/", "\u2044"))
+
```
[the remark this answers is not in the document]
Comment (domfournier) [reply geoh5py/io/h5_reader.py:191]: The `[:]` accesses the numpy.array from the h5py object.

File: geoh5py/io/h5_reader.py (hunk 1 above)
Comment (domfournier) [reply geoh5py/io/h5_reader.py:187]: Has to do with h5py using the "/" as a database seperator, which breaks the structure of the file if we don't replace.

**Review by MatthieuCMira — APPROVED** [review]
I tested it on petro lingo with 1434. 
It works

**Review by MatthieuCMira — CHANGES_REQUESTED** [review]
When pushing back object base to monitoring directory, it flush the values of the selected data. 
We end up with empty data in the original GA workflow.

**Review by MatthieuCMira — APPROVED** [review]

**Comment by MatthieuCMira** [comment]
Tests pass on petro-lingo! 
Well done!

**Pull request merged**